src/test/java/com/plaid/client/integration/depositswitch/DepositSwitchCreateTest.java

```diff
@@ -0,0 +1,57 @@
+package com.plaid.client.integration.depositswitch;
+
+import com.plaid.client.integration.AbstractIntegrationTest;
+import com.plaid.client.request.AccountsGetRequest;
+import com.plaid.client.request.ItemImportRequest;
+import com.plaid.client.request.depositswitch.DepositSwitchCreateRequest;
+import com.plaid.client.response.AccountsGetResponse;
+import com.plaid.client.response.ItemImportResponse;
+import com.plaid.client.response.depositswitch.DepositSwitchCreateResponse;
+
+import org.junit.Test;
+import retrofit2.Response;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+
+public class DepositSwitchCreateTest extends AbstractIntegrationTest {
+  @Test
+  public void testDepositSwitchCreateSuccess() throws Exception {
+    HashMap<String, String> userAuth = new HashMap<>();
```

Comment:
you can use static initializer to make your life slightly easier

Reply:
```
new HashMap<>() {
{ "user_id", "user_good"},
{"auth_token", "pass_good"},
}
```

Reply:
I couldn't get this to work - are you sure this works for HashMap? From some stackoverflow'ing, I think that it works for `Map` but couldn't find example of it working for `HashMap`.

Also, don't think we're using static initialization in other test files